tests/pipeline/test_slice.py

```diff
@@ -0,0 +1,322 @@
+"""
+Tests for slicing pipeline terms.
+"""
+from numpy import arange
+from pandas import (
+    DataFrame,
+    date_range,
+    Int64Index,
+    Timestamp,
+)
+from pandas.util.testing import assert_frame_equal
+
+from zipline.errors import NonWindowSafeInput, UnsupportedPipelineColumn
+from zipline.pipeline import CustomFactor, Pipeline
+from zipline.pipeline.data import USEquityPricing
+from zipline.pipeline.engine import SimplePipelineEngine
+from zipline.pipeline.factors import (
+    Returns,
+    RollingLinearRegressionOfReturns,
+    RollingPearsonOfReturns,
+    RollingSpearmanOfReturns,
+    SimpleMovingAverage,
+)
+from zipline.pipeline.loaders.frame import DataFrameLoader
+from zipline.pipeline.slice import Slice
+from zipline.testing import check_arrays, parameter_space
+from zipline.testing.fixtures import WithTradingEnvironment, ZiplineTestCase
+
+from .test_engine import (
+    AssetID,
+    AssetIDPlusDay,
+    OpenPrice,
+)
```

Comment:
Yes.

On Mon, Jun 13, 2016 at 3:03 PM, David Michalowicz <notifications@github.com
> wrote:

> In tests/pipeline/test_slice.py
> <https://github.com/quantopian/zipline/pull/1267#discussion_r66848199>:
>
> > +    Returns,
> > +    RollingLinearRegressionOfReturns,
> > +    RollingPearsonOfReturns,
> > +    RollingSpearmanOfReturns,
> > +    SimpleMovingAverage,
> > +)
> > +from zipline.pipeline.loaders.frame import DataFrameLoader
> > +from zipline.pipeline.slice import Slice
> > +from zipline.testing import check_arrays, parameter_space
> > +from zipline.testing.fixtures import WithTradingEnvironment, ZiplineTestCase
> > +
> > +from .test_engine import (
> > +    AssetID,
> > +    AssetIDPlusDay,
> > +    OpenPrice,
> > +)
>
> @ssanderson <https://github.com/ssanderson> Is it bad practice to import
> from another test file?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/quantopian/zipline/pull/1267/files/820af991752aaedcd8093e0ad803fba534c963d9#r66848199>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABg8hYMxwNBaW0VG_dOTofs1EqAyReygks5qLamMgaJpZM4IwHT3>
> .
>